external-crates/move/crates/move-compiler/src/sui_mode/linters/freezing_capability.rs

```diff
@@ -39,16 +40,15 @@ pub struct WarnFreezeCapability;
 
 pub struct Context<'a> {
     env: &'a mut CompilationEnv,
-    capability_regex: Regex,
 }
 
+static REGEX: Lazy<Regex> =
+    Lazy::new(|| Regex::new(r".*((Cap[A-Z0-9_]+)|Capability|(Cap$)).*").unwrap());
```

Comment:
```suggestion
    Lazy::new(|| Regex::new(r".*Cap(?:[A-Z0-9_]+|ability|$).*").unwrap());
```
Shorter but less clear Regex.

Reply:
Would it make sense to revise the regex as:
```
.*Cap(?:(?:v)[A-Z0-9_]+|ability|$).*
```
This would catch `Capv0`, etc, as well.

Reply:
I don't think I want to catch `Capv0`, since someone should write it as `CapV0`. In short, I'm trying to find word boundaries